styles/new/base.css

```diff
@@ -41,3 +41,50 @@ ol {
     text-neutral-900
     dark:text-white;
 }
+
+table {
+  @apply w-full 
+  border-separate 
+  border-spacing-0 
+  rounded
+  border
+  border-neutral-200 
+  text-left 
+  dark:border-neutral-800;
+
+  th {
```

Comment:
`th` and `td` share a lot of properties. Can u create a common selector `th, td`? DRY